VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/participant-status/participant-status.component.html

```diff
@@ -0,0 +1,49 @@
+<div [ngClass]="{'yellow': isInCurrentRoom()}" class="text-right">
+    <ng-container *ngIf="!isInCurrentRoom()">
+        <ng-container [ngSwitch]="status">
+            <ng-container *ngSwitchCase="'Calling'">
```

Comment:
could we add this to ConsultationAnswer and reference it here?

Reply:
It would be nice, I did this based on the old implementation. Calling doesn't have any context on the API it's something we handle purely on the UI. Maybe we could map to an Enum defined in the UI that additionally contains Calling so we aren't passing around strings as that is less than ideal. I don't think I will get a chance to do that in this ticket though.